Sakura/Modules/handlers.py

```
@@ -15,117 +15,103 @@
+        if voice_data:
+            await message.reply_voice(voice=voice_data)
+            log_action("INFO", "✅ Voice message response sent successfully", user_info)
         else:
-            await handle_text(update, context)
+            await message.reply_text(ai_response)
+            log_action("INFO", "✅ Text message response sent successfully", user_info)
 
         await log_response(user_id)
         log_action("DEBUG", "⏰ Updated user response time", user_info)
 
     except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)